### PR TITLE
Add movement summary

### PR DESCRIPTION
**Pull request by ellisrichardj**
This change generates a movement summary which can be displayed when users click on a sample in the Nextstrain display.  The summary is generated from the metadata samples table by firstly checking if the CPH in column 6 is the same as Loc0, which should be the birth CPH.  If there are the same it indicates that the animal has not moved off the farm it was born on so `PreviousMovement` is FALSE, and conversely if they are different then the animal has clearly moved at some point, so `PreviousMovement` is TRUE.  The number of movements `MoveCount` is then captured by simply counting the number of non-null entries in the multitude of LocN columns.

### CODE REVIEW

**Review by nick-pestell — COMMENTED** [review]
Looking good, a few small suggestions/comments

File: bin/filterMetadata.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ def filter(metadata_csv):
 
     metadata_df = pd.read_csv(metadata_csv, dtype='object')
     metadata_df.set_index('Submission', inplace=True)
+    move_count = metadata_df.iloc[:, 20::5].notnull().sum(axis=1)
```
[the remark this answers is not in the document]
Comment (ellisrichardj) [reply bin/filterMetadata.py:17]: I have started this from 20 as all samples will have two entries - Loc0 for the birth CPH and then a second one for the slaughterhouse.

I realise we may need to change this as the data structure evolves, but I think its useful to capture this.  The function can be adapted when we no what that looks like

File: bin/filterMetadata.py [hunk 2]
```diff
@@ -24,6 +25,13 @@ def filter(metadata_csv):
     metadata_df.replace(r'^\s*$', np.nan, regex=True, inplace=True)
     metadata_df.fillna('NA', inplace=True)
 
+    # Indicate if there is a history of cattle movement (True/False)
+    metadata_df['PreviousMovement'] = ''
+    metadata_df.loc[(metadata_df['Loc0'] != metadata_df['CPH'], 'PreviousMovement')] = 'True'
+    metadata_df.loc[(metadata_df['Loc0'] == metadata_df['CPH'], 'PreviousMovement')] = 'False'
+    metadata_df.loc[(metadata_df['Loc0'] == 'NA', 'PreviousMovement')] = 'False'
+    metadata_df['MoveCount'] = move_count
```
[the remark this answers is not in the document]
Comment (ellisrichardj) [reply bin/filterMetadata.py:33]: I'll take a look - thanks for the suggestion

**Review by Hira-Naveed — APPROVED** [review]
Nice to see this addition of a movement summary,- great work!

File: bin/filterMetadata.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nick-pestell) [reply bin/filterMetadata.py:37]: Yeah this looks great. I would probably define `moveTF` outside of the `filter` function, I think. But, up to you if you would prefer to keep it like this.

**Review by nick-pestell — APPROVED** [review]
Aprooved

**Pull request merged**